nlpretext/textloader.py

```diff
@@ -104,16 +142,21 @@ def _read_text_parquet(self, files_path):
 
         Returns
         -------
-        dask.dataframe
+        dask.dataframe | pandas.DataFrame
         """
-        text_ddf = dd.read_parquet(files_path)
+        text_ddf = self.loader.read_parquet(files_path, encoding=self.encoding)
         try:
             return text_ddf[[self.text_column]]
         except KeyError:
             raise KeyError(f"Specified text_column '{self.text_column}' not in file keys")
 
     def read_text(
-        self, files_path, file_format=None, encoding=None, compute_to_pandas=True, preprocessor=None
+        self,
+        files_path,
+        file_format=None,
+        encoding=None,
+        compute_to_pandas=True,
+        preprocessor=None,
     ):
```

Comment:
Typehinting on return value

Reply:
Done but dask type hints are not streamlined -> complicated with mypy